typescript/package.json

```diff
@@ -0,0 +1,62 @@
+{
+	"name": "<%= moduleName %>",
+	"version": "0.0.0",
+	"description": "",
+	"license": "MIT",
+	"repository": "<%= githubUsername %>/<%= moduleName %>",
+	"author": {
+		"name": "<%= name %>",
+		"email": "<%= email %>",
+		"url": "<%= website %>"
+	},
+	"engines": {
+		"node": ">=6"
+	},
+	"scripts": {
+		"build": "del dist && tsc",
+		"prepare": "npm run build",
+		"pretest": "npm run build",
+		"test": "xo && ava"
+	},
+	"main": "dist",
+	"types": "dist",
+	"files": [
+		"dist",
+		"!*.test.*"
+	],
+	"keywords": [
+		""
+	],
+	"dependencies": {},
+	"devDependencies": {
+		"@sindresorhus/tsconfig": "^0.2.1",
+		"@types/node": "^11.9.0",
+		"@typescript-eslint/eslint-plugin": "^1.3.0",
+		"ava": "^1.2.1",
+		"del-cli": "^1.1.0",
+		"eslint-config-xo-typescript": "^0.8.0",
+		"ts-node": "^8.0.2",
+		"typescript": "^3.3.3",
+		"xo": "^0.24.0"
+	},
+	"ava": {
+		"babel": false,
+		"compileEnhancements": false,
+		"extensions": [
+			"ts"
+		],
+		"require": [
+			"ts-node/register"
+		]
+	},
+	"xo": {
+		"extends": "xo-typescript",
+		"extensions": [
+			"ts"
+		],
+		"rules": {
+			"@typescript-eslint/promise-function-async": "off",
+			"ava/no-ignored-test-files": "off"
```

Comment:
We need to add TS support here: https://github.com/avajs/eslint-plugin-ava/blob/8ebbca088b091b821788450fd7396ee07986a98c/util.js#L17-L23 Or make it respect the AVA `extensions` option.

You should open an issue there ;)